_posts/2017-09-08-scalafix-v0.5.md

```diff
@@ -0,0 +1,191 @@
+---
+category: blog
+by: Ólafur Páll Geirsson
+title: "Announcing Scalafix v0.5"
+---
+
+I am excited to announce the release of [Scalafix v0.5.0][].
+This release introduces new features in addition to several bug fixes and
+integration improvements.
+
+If you are not familiar with Scalafix, Scalafix is a rewrite and linting tool
+for Scala.
+The project is developed at the [Scala Center][sc] with the mission to help
+automate migration between different Scala compiler and library versions.
+
+> This is the third post on Scalafix and scalameta.
+> You might be interested in reading the previous post
+> [here](http://www.scala-lang.org/blog/2017/02/27/Scalafix-v0.3.html).
+
+
+## Linting
+
+Scalafix v0.5 supports linters.
```

Comment:
Maybe add a sentence explaining what linting is? (Not everybody knows. The only people who seem to know what linters are are the people who are in charge of builds or process)

Reply:
Good idea, added.

Reply:
> Linters are programs that analyze source code for potential bugs.

Reply:
How about,
> Linters are programs that analyze source code for usages of language constructs that could unwittingly lead to bugs, and emit warnings when these potential issues are found. Linters are particularly useful for defining and enforcing coding standards across teams.

(Your previous definition also fits the definition of a compiler. It's too vague.)

Reply:
Very nice, I've updated the text to use your suggestion.